src/main/java/practice/pankaj/Program3.java

```diff
@@ -0,0 +1,29 @@
+package Pankaj;
+import java.util.*;
+
+public class Program3 {
+
+
+    public static void main(String[] args)
+    {
+        int n, max;
+        Scanner s = new Scanner(System.in);
+        System.out.print("Enter number of elements in the array:");
+        n = s.nextInt();
+        int a[] = new int[n];
+        System.out.println("Enter elements of array:");
+        for(int i = 0; i < n; i++)
+        {
+            a[i] = s.nextInt();
+        }
+        max = a[0];
+        for(int i = 0; i < n; i++)
+        {
+            if(max < a[i])
+            {
+                max = a[i];
+            }
```

Comment:
# no need for scanner class, to define or mention int array and find the max number from it.  you can refer the below program.

public class P04_LargestNumberInArray {

    public static void main(String[] args) {
        // assignment - find the largest number in the ages array in unsorted ages
        int[] ages = {22, 34, 18, 17, 28, 3, 9, 12, 16};
        int max = ages[0];
        for (int i = 0; i < ages.length; i++) {
            if (ages[i] > max) {
                max = ages[i];
            }
        }
        System.out.println("max in the array is " + max);
    }

}